services/web/server/src/simcore_service_webserver/application_settings.py

```diff
@@ -375,13 +376,16 @@ def to_client_statics(self) -> dict[str, Any]:
 
 
 def setup_settings(app: web.Application) -> ApplicationSettings:
-    app[APP_SETTINGS_KEY] = settings = ApplicationSettings.create_from_envs()
-    log.info(
+    settings: ApplicationSettings = ApplicationSettings.create_from_envs()
+    app[APP_SETTINGS_KEY] = settings
+    log.debug(
```

Comment:
maybe also apply the `_logger` rename here as well
```suggestion
    _logger.debug(
```